site/en/docs/webstore/register/index.md

```diff
@@ -2,35 +2,41 @@
 layout: "layouts/doc-post.njk"
 title: "Register as a Chrome Web Store developer"
 date: 2020-08-19
+updated: 2020-06-27
 description: How to register as a Chrome Web Store developer.
 ---
 
 Before you can publish items on the Chrome Web Store, you must register as a CWS developer and pay a
 one-time registration fee. You must provide a developer email when you create your developer
 account; here are some tips about which email to use:
 
-* It's a good idea to decouple your personal account and your Chrome Web Store publishing account.
-  That's why we suggest using a new email account just for publishing your Chrome Web Store items.
-* If you recently deleted your Chrome Web Store developer account, you can't reuse its associated
+* Because you will receive important emails about your extension or you may want to delete one of your accounts, we suggest using a new email account just for publishing your Chrome Web Store items. 
+* Make sure to check this address frequently so that you're aware of any important alerts or
+  announcements.
+* If you recently [deleted your Chrome Web Store developer account][1], you can't reuse its associated
   email identity to create a new one. 
   {% Aside 'warning' %}
-  If you requested the deletion of your account by mistake, please contact developer support
+  If you requested the deletion of your account by mistake, please [contact developer support][2]
   immediately.
   {% endAside %}
-* Make sure to check this address frequently so that you're aware of any important alerts or
-  announcements.
 
-
-
-To register, just access the [developer console][1]. The first time you do this, a registration
-screen appears as shown here:
+To register, just access the [developer console][3]. The first time you do this, the following registration screen will appear. First agree to the developer agreement and policies then pay the registration fee.
 
 {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/adH2VFqS92VdzPTjsHtB.png", 
        alt="Screenshot of the Chrome Web Store developer registration page", height="678", width="800" %}
 
-Once you pay the registration fee and agree to the terms, you will not see this registration page
-again.
+Once you pay the registration fee and agree to the terms, you will not see this registration page again.
+
+## Next steps {: #next-steps }
+
+1.  [Create your item's zip file][4]
+2.  [Upload your item to the Chrome Web  Store][5]
+3.  [Set up a group publisher][6] to share the ownership of your extension
```

Comment:
In Markdown lists auto-number, so I prefer the convention:

```
1. Foo
1. Bar
1. Baz
```

Also, this group publisher step is not a next step for many (most?) developers. Perhaps "if you want to share..."

Reply:
Done!